teuthology/task/install/rpm.py

```diff
@@ -177,6 +214,8 @@ def _update_package_list_and_install(ctx, remote, rpm, config):
         if dist_release in ['opensuse', 'sle']:
             _zypper_wipe_all_repos(remote)
             _zypper_addrepo(remote, repos)
+        if dist_release in ['rhel', 'fedora', 'centos']:
```

Comment:
```suggestion
        elif dist_release in ['rhel', 'fedora', 'centos']:
```

otherwise this breaks openSUSE and SLE support.

Reply:
Will fix it.